### PR TITLE
E2e test improvements

### PR DESCRIPTION
**Pull request by krokhmaliuk-virtru**
2. Split large test to smaller
3. Replaced old $$ to locators
4. Fixed failed clean up for 'delete autority test'
5. Added describe steps for better consistency in reports/debug
6. Fixed 404 error
7. Added wait seletors for some tests

### CODE REVIEW

File: abacus-integration-test/e2e/Attributes.spec.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ test.describe('<Attributes/>', () => {
     });
   });
 
-  test.afterEach(async ({ authority}) => {
+  test.afterEach(async ({ authority, page}, testInfo) => {
```
Comment (sievdokymov-virtru) [inline abacus-integration-test/e2e/Attributes.spec.ts:40]: What this change is used for?

File: abacus-integration-test/e2e/Attributes.spec.ts [hunk 2]
```diff
@@ -222,58 +206,123 @@ test.describe('<Attributes/>', () => {
...
     })
 
+    await test.step('Cleanup', async () => {
+      await deleteAttributeViaAPI(apiContext, authority, firstAttributeName,['A', 'G', 'H'], "anyOf")
+      await deleteAttributeViaAPI(apiContext, authority, secondAttributeName,['C', 'G', 'H'], "allOf")
+      await deleteAttributeViaAPI(apiContext, authority, thirdAttributeName,['B', 'G', 'H'], "hierarchy")
+    })
+  });
+
+  test('should sort attributes by rule, values_array', async ({ page, authority}) => {
```
Comment (sievdokymov-virtru) [inline abacus-integration-test/e2e/Attributes.spec.ts:246]: by ID and values_array, not by rule

File: abacus-integration-test/e2e/Attributes.spec.ts [hunk 3]
```diff
@@ -222,58 +206,123 @@ test.describe('<Attributes/>', () => {
...
+    const sortByToolbarButton = selectors.attributesPage.attributesHeader.sortByToolbarButton
```
Comment (sievdokymov-virtru) [inline abacus-integration-test/e2e/Attributes.spec.ts:247]: lot of duplicated code after division. Do we really need it? Doesn't increasing test timeout solve time-to-time failures?

File: abacus-integration-test/e2e/Attributes.spec.ts (hunk 1 above)
Comment (krokhmaliuk-virtru) [reply abacus-integration-test/e2e/Attributes.spec.ts:40]: not used anymore

**Review by sievdokymov-virtru — APPROVED** [review]

**Pull request merged**